tests/main/interfaces-snap-interfaces-requests-control/task.yaml

```diff
@@ -0,0 +1,90 @@
+summary: Verify that snap-interfaces-requests-control grants read-access to relevant endpoints
+details: |
+    The snap-interfaces-requests-control interface grants access to prompting clients
+    (e.g. prompting-client) to a subset of snapd's API endpoints needed
+    to view and manage request prompts and request rules.
+
+    Specifically:
+      - /v2/notices: to read change-update and refresh-inhibit notices
+      - /v2/system-info: to check whether prompting is supported/enabled
+      # TODO: - /v2/interfaces/requests/prompts: to receive and reply to request prompts
+      # TODO: - /v2/interfaces/requests/rules: to view and manage request rules
+
+environment:
+    # not all terminals support UTF-8, but Python tries to be smart and attempts
+    # to guess the encoding as if the output would go to the terminal, but in
+    # fact all the test does is pipe the output to jq
+    PYTHONIOENCODING: utf-8
+
+prepare: |
+    snap install --edge jq
```

Comment:
Is edge relevant here?

Don't we have test-snapd-jq somewhere?

Reply:
Seems like we use a mix of standard `jq`, `--devmode`, and `--edge` elsewhere in snapd. I don't believe the `--edge` is relevant here, I'll remove it and test.

```
$ grep -r test-snapd-jq                 
tests/lib/snaps/store/test-snapd-jq-core24/snapcraft.yaml:name: test-snapd-jq-core24
tests/lib/prepare.sh:        snap install --devmode --edge test-snapd-jq-core24
tests/lib/prepare.sh:        snap alias test-snapd-jq-core24.jq jq
tests/lib/prepare.sh:    snap remove --purge test-snapd-jq-core24
```

We maybe could use `test-snapd-jq-core24`, but we use standard `jq` (from mvo) in most places, so I'd rather stick to it.